### PR TITLE
Run Prettier on JS code fences, part 8

### PR DESCRIPTION
**Pull request by OnkarRuikar**
The PR focuses only on JS code fences.

Idea is to gradually prettify all the JS code fences before the full automation.

cc/ @Josh-Cena , @wbamberg

### CODE REVIEW

File: files/en-us/webassembly/javascript_interface/module/imports/index.md [hunk 1]
```diff
@@ -48,8 +48,7 @@ See imports.html [source code](https://github.com/mdn/webassembly-examples/blob/
 [live version](https://mdn.github.io/webassembly-examples/js-api-examples/imports.html).
 
 ```js
-WebAssembly.compileStreaming(fetch('simple.wasm'))
-.then(function(mod) {
+WebAssembly.compileStreaming(fetch("simple.wasm")).then(function (mod) {
```
Comment (Josh-Cena) [inline files/en-us/webassembly/javascript_interface/module/imports/index.md:51]:
```suggestion
WebAssembly.compileStreaming(fetch("simple.wasm")).then((mod) => {
```

File: files/en-us/webassembly/javascript_interface/validate/index.md [hunk 2]
```diff
@@ -50,12 +50,14 @@ fetches a .wasm module and converts it into a typed array.
...
 ```js
-fetch('simple.wasm').then((response) =>
-  response.arrayBuffer()
-).then(function(bytes) {
-  const valid = WebAssembly.validate(bytes);
-  console.log(`The given bytes are ${valid ? "" : "not "}a valid wasm module`);
-});
+fetch("simple.wasm")
+  .then((response) => response.arrayBuffer())
+  .then(function (bytes) {
```
Comment (Josh-Cena) [inline files/en-us/webassembly/javascript_interface/validate/index.md:55]:
```suggestion
  .then((bytes) => {
```

File: files/en-us/webassembly/javascript_interface/table/set/index.md [hunk 3]
```diff
@@ -80,12 +80,13 @@ adds two function references to the table, both of which print out
...
 
 ```js
-WebAssembly.instantiateStreaming(fetch('table2.wasm'), importObject)
-.then(function(obj) {
-  console.log(tbl.length);
-  console.log(tbl.get(0)());
-  console.log(tbl.get(1)());
-});
+WebAssembly.instantiateStreaming(fetch("table2.wasm"), importObject).then(
+  function (obj) {
```
Comment (Josh-Cena) [inline files/en-us/webassembly/javascript_interface/table/set/index.md:84]:
```suggestion
  (obj) => {
```

**Review by bsmth — APPROVED** [review]
Looks good to me 👍🏻

**Review by Josh-Cena — APPROVED** [review]

**Pull request merged**